torch2jax/__init__.py

```diff
@@ -157,11 +160,22 @@ def pow(*args, **kwargs): return torch.pow(*args, **kwargs)
   def size(self): return self.shape
   def sum(*args, **kwargs): return torch.sum(*args, **kwargs)
   def transpose(*args, **kwargs): return torch.transpose(*args, **kwargs)
-  def view(self, *shape): return Torchish(jnp.reshape(self.value, shape))
-  reshape = view
   def unbind(*args, **kwargs): return torch.unbind(*args, **kwargs)
+  def sin(*args, **kwargs): return torch.sin(*args, **kwargs)
+  def cos(*args, **kwargs): return torch.cos(*args, **kwargs)
+  def unsqueeze(self, dim): return Torchish(jnp.expand_dims(self.value, axis=dim))
+  def float(self): return Torchish(jnp.astype(self.value, jnp.float32))
+  def to(self, *args, **kwargs): return self  # ignore device movement, jax manages its own placement.
+  def contiguous(self): return self
   # fmt: on
 
+  def view(self, *shape):
+    if len(shape) == 1 and isinstance(shape[0], Sequence):
+      shape = shape[0]
```

Comment:
let's add a test case for this. you can add it here: https://github.com/samuela/torch2jax/blob/7132ab7d7354a12afed82902581180de143aabf3/tests/test_core.py#L118-L122

Reply:
Sure, what is the local configuration when you run the test? I figure that the test wouldn't run unless I turn the GPU off. Complaining about `tensor.numpy()` should be `tensor.cpu().numpy()`. Do we want to add an ci script to github?